src/ideas/decorations/SlotMachine/idea.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "ac5855ff-d070-43a3-a29e-d18c66dbbac8",
+  "name": "Slot Machine",
+  "purpose": "Open a gambling hall.",
+  "schema": [
+    {
+      "name": "position",
+      "type": "position"
+    },
+    {
+      "name": "rotation",
+      "type": "rotation"
+    },
+    {
+      "name": "width",
+      "type": "number"
+    },
+    {
+      "name": "height",
+      "type": "number"
+    },
+    {
+      "name": "num_of_icons",
+      "type": "number"
+    },
+    {
+      "name": "num_plays_per_day",
```

Comment:
would suggest `playsPerDay`, there's not too much space on builder wand title currently and i have logic to convert camel case to lowercase + spaced out, i.e. "playsPerDay" --> "plays per day"